selenium/exercise-6.rb

```diff
@@ -0,0 +1,18 @@
+require 'selenium-webdriver'
+
+Selenium::WebDriver::Chrome::Service.driver_path = "C:/Users/anup.desai/Desktop/infuse/Practice-ANUPDESAI/selenium/geckodriver.exe"
+
+driver = Selenium::WebDriver.for :firefox
+
+driver.get "https://letcode.in/forms"
+
+element = driver.find_element(:css, "div:nth-child(2).columns.container > div:last-child.column.is-half > div.field > div.control > div.select")
+
+element = driver.find_elements(:tag_name,"option")
+
```

Comment:
A better approach is to use

`dropdown = Selenium::WebDriver::Support::Select.new(dropdown_xpath_element)`

You should make use of the functions mentioned here: https://www.selenium.dev/selenium/docs/api/rb/Selenium/WebDriver/Support/Select.html